packages/apps/dev-wallet/src/pages/create-profile/create-profile.css.ts

```diff
@@ -0,0 +1,13 @@
+import { style } from '@vanilla-extract/css';
+
+export const buttonLinkClass = style([
+  {
+    textDecoration: 'none',
```

Comment:
will be not needed in case of handling submit button as logic and not as link inside a button